packages/runtime/lib/utils.js

```diff
@@ -35,10 +39,17 @@ function loadConfig (minimistConfig, args, overrides, replaceEnv = true) {
   return pltConfigLoadConfig(minimistConfig, args, store, overrides, replaceEnv)
 }
 
+function loadEmptyConfig (path, overrides, replaceEnv = true) {
+  const store = new Store()
+  store.add(platformaticRuntime)
```

Comment:
I don't think the runtime should be there if it's empty.

Reply:
Good point. Removing it.